Example/common.js

```diff
@@ -35,3 +33,13 @@ Praesent ac risus nec augue commodo semper eu eget quam. Donec aliquam sodales c
 
 Fusce interdum nisi a risus aliquet, non dictum metus cursus. Praesent imperdiet sapien orci, quis sodales metus aliquet id. Aliquam convallis pharetra erat. Fusce gravida diam ut tellus elementum sodales. Fusce varius congue neque, quis laoreet sapien blandit vestibulum. Donec congue libero sapien, nec varius risus viverra ut. Quisque eu maximus magna. Phasellus tortor nisi, tincidunt vitae dignissim nec, interdum vel mi. Ut accumsan urna finibus posuere mattis.
 `;
+
+export const Alert = {
```

Comment:
We only use `Alert` to notify that some action has been triggered. I checked around our example app codebase and we never use non-standard options like buttons and alert types. I'd recommend that instead of adding this to commons we just use `alert` method that is globally available in RN on web and mobile

Reply:
Sure, but I still kept it in `common.js` in case we want to differentiate the behavior in the future.